sdk/tables/data-tables/src/generated/generatedClientContext.ts

```diff
@@ -6,13 +6,11 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
-import * as coreHttp from "@azure/core-http";
+import { ServiceClient } from "@azure/core-client";
+import { stringifyXML, parseXML } from "@azure/core-xml";
 import { GeneratedClientOptionalParams } from "./models";
 
-const packageName = "@azure/data-tables";
-const packageVersion = "1.0.0-beta.3";
```

Comment:
hmm, why?

Reply:
This can be left in the codegen, I think I just took it out because the convenience clients always provide this info so it's technically not used:

https://github.com/Azure/azure-sdk-for-js/blob/ac4f023ffc601366c81bc4f3d3a057d210f94065/sdk/tables/data-tables/src/TableClient.ts#L139

Tactically, I think I omitted it because I don't have `getDefaultUserAgentValue()` exposed right now, though perhaps we should add that back.